controlpanel/api/models/parameter.py

```diff
@@ -0,0 +1,81 @@
+from django.conf import settings
+from django.core.validators import RegexValidator
+from django.db import models
+from django.db.models.signals import pre_delete, pre_save
+from django.dispatch import receiver
+
+from django_extensions.db.models import TimeStampedModel
+
+from controlpanel.api.aws import aws, arn
+
+
+APP_TYPE_CHOICES = (
+    ('airflow', 'Airflow'),
+    ('webapp', 'Web app'),
+)
+
+
+class Parameter(TimeStampedModel):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._value = None
+
+    @property
+    def value(self):
+        return self._value
+
+    @value.setter
+    def value(self, val):
+        self._value = val
```

Comment:
Why is `value` a `@property`?

Reply:
We don't want to save it. Also we want to be able to show and set it in the API then send the value to AWS on save.

Reply:
Yeah, but the property doesn't do anything - you could just have
```python
def __init__(self, *args, **kwargs):
    super().__init__(*args, **kwargs)
    self.value = None
```
couldn't you?